src/views/AmalgamationRegular/Information.vue

```diff
@@ -2,7 +2,7 @@
   <div id="amalgamation-regular-information">
     <!-- Amalgamating Businesses -->
     <section class="mt-10">
-      <header id="name-header">
+      <header id="amalgamating-businesses">
```

Comment:
For validation handling -- see `validComponents` below and `scrollToInvalidComponent ()`.